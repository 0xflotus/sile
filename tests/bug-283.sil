\begin[papersize=a6]{document}
\font[family=Libertinus Serif,size=9pt,language=tr]
Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın

Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın

Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının
\end{document}