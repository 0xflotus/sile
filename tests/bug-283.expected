<tests/bug-283.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	40.9764
My 	38.8443
Set font 	Linux Libertine O;12.7;400;;normal;;LTR
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	101.7617
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	154.0635
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	214.8489
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	267.1506
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	327.9360
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	380.2378
T	51 66	(Ra)
Mx 	393.4966
T	14	(-)
Mx 	20.9764
My 	54.0843
T	67	(b)
Mx 	27.2375
T	8	(')
Mx 	29.6505
T	67 74	(bi)
Mx 	39.3533
T	78 74 91 70	(mize)
Mx 	67.8633
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	119.2714
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	179.1631
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	230.5712
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	290.4630
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	341.8711
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	20.9764
My 	69.3243
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	73.7836
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	135.0744
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	187.8817
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	249.1725
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	301.9798
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	363.2706
T	53 66 79	(Tan)
Mx 	382.9175
T	83 241	(rı)
Mx 	391.0836
T	8	(')
Mx 	393.4966
T	14	(-)
Mx 	20.9764
My 	84.5643
T	79 241 79	(nın)
Mx 	42.7556
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	103.2444
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	155.2497
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	215.7385
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	267.7437
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	328.2326
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	380.2378
T	51 66	(Ra)
Mx 	393.4966
T	14	(-)
Mx 	20.9764
My 	99.8043
T	67	(b)
Mx 	27.2375
T	8	(')
Mx 	29.6505
T	67 74	(bi)
Mx 	39.3533
T	78 74 91 70	(mize)
Mx 	67.8633
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	119.2714
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	179.1631
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	230.5712
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	290.4630
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	341.8711
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	20.9764
My 	115.0443
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	71.6310
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	130.7691
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	181.4237
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	240.5619
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	291.2165
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	350.3547
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	20.9764
My 	130.2843
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	82.0323
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	134.6045
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	195.6604
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	248.2327
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	309.2886
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	361.8609
T	51 66	(Ra)
Mx 	375.1197
T	67	(b)
Mx 	381.3808
T	8	(')
Mx 	383.7938
T	67 74	(bi)
Mx 	393.4966
T	14	(-)
Mx 	20.9764
My 	145.5243
T	78 74 91 70	(mize)
Mx 	49.3530
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	100.6278
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	160.3861
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	40.9764
My 	160.7643
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	100.7182
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	151.9765
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	211.7183
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	262.9766
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	322.7184
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	373.9767
T	51 66	(Ra)
Mx 	387.2355
T	67	(b)
Mx 	393.4966
T	14	(-)
Mx 	20.9764
My 	176.0043
T	1757	(’)
Mx 	24.3800
T	67 74	(bi)
Mx 	34.0828
T	78 74 91 70	(mize)
Mx 	62.4806
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	114.7671
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	175.5373
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	227.8238
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	288.5940
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	340.8805
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	20.9764
My 	191.2443
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	73.6185
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	134.7442
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	187.3864
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	248.5121
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	301.1543
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	362.2800
T	53 66 79	(Tan)
Mx 	381.9269
T	83 241	(rı)
Mx 	390.0930
T	1757	(’)
Mx 	393.4966
T	14	(-)
Mx 	20.9764
My 	206.4843
T	79 241 79	(nın)
Mx 	41.0121
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	100.7480
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	152.0003
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	211.7362
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	262.9885
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	322.7244
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	373.9767
T	51 66	(Ra)
Mx 	387.2355
T	67	(b)
Mx 	393.4966
T	14	(-)
Mx 	20.9764
My 	221.7243
T	1757	(’)
Mx 	24.3800
T	67 74	(bi)
Mx 	34.0828
T	78 74 91 70	(mize)
Mx 	62.4806
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	114.7671
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	175.5373
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	227.8238
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	288.5940
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	340.8805
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	20.9764
My 	236.9643
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	73.6185
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	134.7442
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	187.3864
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	248.5121
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	301.1543
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	362.2800
T	53 66 79	(Tan)
Mx 	381.9269
T	83 241	(rı)
Mx 	390.0930
T	1757	(’)
Mx 	393.4966
T	14	(-)
Mx 	20.9764
My 	252.2043
T	79 241 79	(nın)
Mx 	41.9065
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	102.5369
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	154.6836
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	215.3140
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	267.4607
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	328.0910
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	380.2378
T	51 66	(Ra)
Mx 	393.4966
T	14	(-)
Mx 	20.9764
My 	267.4443
T	67	(b)
Mx 	27.2375
T	1757	(’)
Mx 	30.6411
T	67 74	(bi)
Mx 	40.3439
T	78 74 91 70	(mize)
Mx 	68.7170
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	120.9788
T	51 66 67 1757 67 74 78 74 91 70	(Rab’bimize)
Mx 	181.7242
T	53 66 79 83 241 1757 79 241 79	(Tanrı’nın)
Mx 	40.9764
My 	282.6843
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	99.3551
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	148.9962
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	207.3749
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	257.0160
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	315.3948
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	365.0359
T	51 66 67	(Rab)
Mx 	384.5558
T	67 74	(bi)
Mx 	394.2586
T	14	(-)
Mx 	20.9764
My 	297.9243
T	78 74 91 70	(mize)
Mx 	49.0258
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	97.5603
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	154.8324
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	203.3669
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	260.6390
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	309.1735
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	366.4456
T	53 66 79	(Tan)
Mx 	386.0925
T	83 241	(rı)
Mx 	394.2586
T	14	(-)
Mx 	20.9764
My 	313.1643
T	79 241 79	(nın)
Mx 	42.5433
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	100.6609
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	150.0409
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	208.1584
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	257.5384
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	315.6559
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	365.0359
T	51 66 67	(Rab)
Mx 	384.5558
T	67 74	(bi)
Mx 	394.2586
T	14	(-)
Mx 	20.9764
My 	328.4043
T	78 74 91 70	(mize)
Mx 	49.0258
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	97.5603
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	154.8324
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	203.3669
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	260.6390
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	309.1735
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	366.4456
T	53 66 79	(Tan)
Mx 	386.0925
T	83 241	(rı)
Mx 	394.2586
T	14	(-)
Mx 	20.9764
My 	343.6443
T	79 241 79	(nın)
Mx 	42.5433
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	100.6609
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	150.0409
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	208.1584
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	257.5384
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	315.6559
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	365.0359
T	51 66 67	(Rab)
Mx 	384.5558
T	67 74	(bi)
Mx 	394.2586
T	14	(-)
Mx 	20.9764
My 	358.8843
T	78 74 91 70	(mize)
Mx 	49.0258
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	97.5603
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	154.8324
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	203.3669
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	260.6390
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	309.1735
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	366.4456
T	53 66 79	(Tan)
Mx 	386.0925
T	83 241	(rı)
Mx 	394.2586
T	14	(-)
Mx 	20.9764
My 	374.1243
T	79 241 79	(nın)
Mx 	42.5433
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	100.6609
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	150.0409
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	208.1584
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	257.5384
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	315.6559
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	365.0359
T	51 66 67	(Rab)
Mx 	384.5558
T	67 74	(bi)
Mx 	394.2586
T	14	(-)
Mx 	20.9764
My 	389.3643
T	78 74 91 70	(mize)
Mx 	49.3711
T	53 66 79 83 241 79 241 79	(Tanrının)
[1] Mx 	207.4176
My 	553.7327
Set font 	Gentium Plus;10;400;;normal;;LTR
T	20	(1)
End page
Finish

